pycparser/_ast_gen.py

```diff
@@ -187,11 +189,38 @@ def _gen_attr_names(self):
 _PROLOGUE_CODE = r'''
 import sys
 
+def _repr(obj):
+    """
+    Get the representation of an object, with dedicated pprint-like format for lists and tuples.
+    """
+    if isinstance(obj, list):
```

Comment:
Why only list if the comment says tuples too?

Reply:
My first implementation handled `tuple`s too.
Then I realized that only `list`s were created by `c_parser`, and that I only ever used `list`s too.
So I dropped the `tuple`-specific code (but forgot to remove the comment).